modules/node_modules/@ncigdc/components/Project.js

```diff
@@ -119,22 +105,33 @@ const Project = ({
         <DownloadButton
           disabled={!biospecimenCount}
```

Comment:
👍 for see any issues with me making that global on the Button component? so if you pass in `disabled` it will disable hover and change it to grey?